style.css

```diff
@@ -237,13 +237,27 @@ img {
   }
   nav {
     width: 100%;
+    /* position: relative; */
```

Comment:
comment?

Reply:
I was testing the css and I forgot to delete the comment.